src/Augurk.Test/Managers/FeatureManagerTests.cs

```diff
@@ -447,49 +384,6 @@ public async Task CanUpdateExistingFeature()
             }
         }
 
-        /// <summary>
-        /// Tests that the <see cref="FeatureManager" /> class sets the expiration on a feature if that is configured.
-        /// </summary>
-        [Fact]
-        public async Task SetsExpirationIfEnabledWhenUpdatingExistingFeature()
```

Comment:
Was het misschien handiger geweest om de tests mbt expiration te ignoren ipv compleet weg te halen?

Reply:
Removed it because we need to rewrite the tests as well. The current implementation of the test assumes the file is removed from the storage, something that with the new data structure will likely not happen. At most we will remove some versionnumbers from within a file. As such, I feel it's best to reimplement the feature from scratch, perhaps even rethinking how it functionally works.